.github/workflows/internal-images.yml

```diff
@@ -61,28 +55,30 @@ jobs:
           password: ${{ secrets.GHCR_TOKEN }}
 
       # We need QEMU and Buildx for multi-platform (amd64+arm64) image push. See RATIONALE.md
-      - name: Setup QEMU
+      - name: "Setup QEMU"
         uses: docker/setup-qemu-action@v1
 
-      - name: Setup Buildx
+      - name: "Setup Buildx"
         uses: docker/setup-buildx-action@v1
 
-      # This resolves GOROOT_NAME (ex GOROOT_1_17_X64) into GO_MINOR_REVISION (ex 1.17.1), by
-      # parsing the directory path GOROOT_NAME evaluates to.
-      #
-      # Ex. Given ${!GOROOT_NAME}=/opt/hostedtoolcache/go/1.17.1/x64 -> GO_MINOR_REVISION=1.17.1
-      - name: Export Go minor revision
-        run: |
-          rev=$(echo ${!GOROOT_NAME}|cut -d/ -f5)
+      - name: "Checkout"
+        uses: actions/checkout@v2
+
+      - name: "Configure Go"
+        run: .github/workflows/configure_go.sh
+
+      - name: "Export Go minor revision"
+        run: |  # Ex. GOROOT=/opt/hostedtoolcache/go/1.17.1/x64 -> GO_MINOR_REVISION=1.17.1
+          rev=$(echo ${GOROOT}|cut -d/ -f5)
           echo "GO_MINOR_REVISION=${rev}" >> $GITHUB_ENV
```

Comment:
I think `GO_MINOR_REVISION` is the same as `GO_VERSION` exported in .github/workflows/configure_go.sh? So I think we can delete this "Export Go minor revision" step

Reply:
it isn't really the same GO_VERSION is 1.17 and REVISION is 1.17.1 I was using go vocabulary from https://golang.org/doc/devel/release though confusingly go calls versions like 1.17 "major Go release" even if it is the minor version component updated!

another way could be GO_RELEASE and GO_REVISION?

Reply:
oh yeah sorry about that. as-is make sense!